eng/testing/xunit/xunit.console.targets

```diff
@@ -57,7 +57,7 @@
   </Target>
 
   <!-- ResolveAssemblyReferences is the target that populates ReferenceCopyLocalPaths which is what is copied to output directory. -->
-  <Target Name="CopyRunnerToOutputDirectory" BeforeTargets="ResolveAssemblyReferences">
+  <Target Name="CopyRunnerToOutputDirectory" BeforeTargets="ResolveAssemblyReferences" Condition="'$(TargetsMobile)' != 'true' or '$(EnableAggressiveTrimming)' != 'true'">
```

Comment:
Isn't PublishingTestsRun better condition to use?

Reply:
The intent was to directly tie the change `Condition="'$(TargetsMobile)' != 'true'` with the flag `EnableAggressiveTrimming`, so that if `EnableAgressiveTrimming` were false, the original functionality of this target (not depending on `TargetsMobile`) would remain. 

It seems like `PublishingTestsRun` is directly tied to `EnableAggressiveTrimming` for now, but I thought it would be more direct to use the flag itself in case `PublishingTestsRun` condition changed.

Reply:
I think we should not invent another vertical condition. The tests should work if you do dotnet publish regardless of trimming configuration.

Reply:
Reverted the change that added `EnableAggressiveTrimming` here.